### DISCUSSION TITLE
Using a Headless remote

### DISCUSSION THREAD

**Discussion by link-leo** [rustdesk/rustdesk#7683] [Q&A]
I have a Windows 10 device on which Rust Desk works great, but only when a monitor is plugged in or I am using Microsoft's Remote Desktop app *in addition* to create a display. Without (while operating headless), Rust Desk returns a "No Display" error when I connect.

I have made this work before while using AnyDesk by attaching a [VGA Dummy Plug,Headless Ghost, Display Emulator](https://www.amazon.com/gp/product/B075ZMVGQS) but Rust Desk doesn't seem to accept that configuration.

Is there a way I can make Rust Desk work on a Headless remote?

**Comment by sorcer1122** [comment]
Having the same issue. The problem is if you are trying to connect to VPS, there is no way to use dummy screen emulator, so hope there is a solution.

**Comment by rustdesk** [comment]
Win: https://www.reddit.com/r/rustdesk/comments/1ja7y90/headless_win_10/
Linux: https://github.com/rustdesk/rustdesk/wiki/Headless-Linux-Support

**Reply by deadbyrd** [reply]
Thank you!
